doc/dev/release_checklist.md

```diff
@@ -132,6 +148,7 @@ We'll use `v4.7.0-rc1` as the intended release version in this example.
 
 - Decide which nightly release you want to turn into a release candidate.
   We will use `nightly-2024-02-29` in this example.
+- It is essential to choose the nightly that will become the release candidate as early as possible, to avoid confusion.
```

Comment:
```suggestion
- It is essential to choose the nightly that will become the release candidate as early as possible, to avoid confusion.
- Through this process you can use `script/release_checklist.py v4.7.0-rc1` to track progress.
```